src/SFA.DAS.Roatp.CourseManagement/Application/Standards/Commands/UpdateApprovedByRegulator/UpdateApprovedByRegulatorCommand.cs

```diff
@@ -0,0 +1,12 @@
+﻿using MediatR;
+using System.Net;
+
+namespace SFA.DAS.Roatp.CourseManagement.Application.Standards.Commands.UpdateApprovedByRegulator
+{
+    public class UpdateApprovedByRegulatorCommand : IRequest<HttpStatusCode>
+    {
+        public int Ukprn { get; set; }
+        public int LarsCode { get; set; }
+        public bool? IsApprovedByRegulator { get; set; }
```

Comment:
This cannot be null. As the user is trying to update the value, we should never allow null here.